### PR TITLE
Add documentation for --keep-missing option

### PR DESCRIPTION
**Pull request by jamacku**

### CODE REVIEW

File: alternatives.8 [hunk 1]
```diff
@@ -266,6 +266,10 @@ Tell which version of
 .B alternatives
 this is (and give some usage information).
 .TP
+.B --keep-missing
+Allow user to keep files which aren't available in newer version of program (e.g. java).
+It prevents issues with missing files due to switching bettwen versions.
```
Comment (lnykryn) [inline alternatives.8]: bettwen -> between

File: alternatives.8 [hunk 2]
```diff
@@ -266,6 +266,10 @@ Tell which version of
 .B alternatives
 this is (and give some usage information).
 .TP
+.B --keep-missing
+Allow user to keep files which aren't available in newer version of program (e.g. java).
```
Comment (lnykryn) [inline alternatives.8]: I would say something like:
When switching between alternatives, if the new variant does not provide some files, keep the links pointed to the previous implementation. 
And I would remove the java reference.

File: alternatives.8 (hunk 1 above)
Comment (jamacku) [reply alternatives.8]: Thank you, I overlooked this one.

File: alternatives.8 (hunk 2 above)
Comment (jamacku) [reply alternatives.8]: Thank you.

**Pull request merged**